src/components/GoalForm/GoalForm.tsx

```diff
@@ -137,16 +126,19 @@ export const GoalForm: React.FC<GoalFormProps> = ({
     return (
         <ModalContent {...attrs}>
             <Form onSubmit={handleSubmit(onSubmit)}>
-                <FormControl flat="bottom" size="l">
+                <FormControl>
                     <FormControlInput
                         {...register('title')}
                         disabled={busy}
                         autoFocus
                         placeholder={tr("Goal's title")}
+                        outline={false}
```

Comment:
Do not use negative props. Use them for extending, like `autoFocus` a few rows above.